docs/user-guide/command-line-interface.md

```diff
@@ -27,56 +27,53 @@ Please note that when passing a glob as a parameter, it will be expanded by your
 The command line utility has several options. You can view the options by running `eslint -h`.
 
 ```text
+eslint [options] file.js [file.js] [dir]
```

Comment:
@not-an-aardvark True, but I think that's beyond the scope of the PR. I'm just trying to sync up the docs with the current `eslint --help` output-- happy to tweak that in a separate PR.

Personally I think it would be more precise to say that at least one file and/or directory and/or glob is required (except when running --help or --version or --print-config), and have placeholders to reflect that fact at the top of the --help output.